ui/src/app/blueprint-overrides.scss

```diff
@@ -35,3 +36,69 @@
     background: transparent;
   }
 }
+.bp3-button, .bp3-menu-item {
+  @include rtl(text-align, left, right);
+
+  > * {
+    @include rtlSupportInvertedProp(margin, left, inherit, 7px);
+    // @include rtl(margin-right, 7px, inherit);
+  }
+}
+.bp3-menu {
+  @include rtl(text-align, left, right);
+}
+
+table {
+  tr td, tr th {
+    @include rtl(text-align, left, right);
+    &.numeric, &.number {
+      @include rtl(text-align, right, left);
+    }
+    // Float on the right, so that numbers overflow to the left.
+    // (FormattedNumber creates a <span>)
+    & > span {
```

Comment:
I think this float rule needs to only apply to `.numeric` and `.number`, not to all nested `span` elements

Reply:
Right!
this basically meant all nested span elements within a td. This was an already existing rule we had.
I moved it back to tables.scss now and I specify a span with a class for the <FormattedNumber> case